include/seqan3/argument_parser/validators.hpp

```diff
@@ -444,10 +444,13 @@ class file_validator_base
         file_guard.remove();
     }
 
-    //!\brief Returns the information of valid file formats.
+    //!\brief Returns the information of valid file extensions.
     std::string valid_extensions_help_page_message() const
     {
-        return detail::to_string("Valid file formats are: [", extensions | views::join(std::string{", "}), "].");
+        if (extensions.empty())
+            return "";
+        else
+            return detail::to_string("Valid file extensions are: [", extensions | views::join(std::string{", "}), "].");
```

Comment:
```suggestion
            return detail::to_string("Valid file extensions are: [", extensions | std::views::join(std::string{", "}), "].");
```

And check that there is no seqan3 views join include.
Regarding `detail::to_string`. Isn't this part of the dependencies @marehr tried to remove from the argument parser? This could be written as `str1 + str2 + str3`

Reply:
The `detail::to_string` is still present in many parts of the code. He sais this one is an open ToDo and he might reimplement the function so using it here won't make a difference?

Reply:
This one is tough, because the `seqan3::views::join` isn't the same as the `std::view::join`.

I had to warkaround it, but it is okay for me to use `seqan3::views::join` here. Also keep the `detail::to_string`.

In this case it is fine as it is.